### ISSUE TITLE
Feature request: making the DTA! Tools menu item optional

### ISSUE THREAD

**Issue by lkintact** [downthemall/downthemall#338]
**Is your feature request related to a problem? Please describe.**
I never run DTA! from the Tools menu, and when I open the menu, the DTA! item stands out too much as it's the only one that has an icon, which makes it somewhat distracting (circled in red in image below).

![DTA!_Tools_menu_item](https://user-images.githubusercontent.com/24726299/119385324-a4da8000-bcce-11eb-960b-4b7068aaf194.png)

**Describe the solution you'd like**
Having an option to disable DTA! Tools menu item.

DownThemAll! version: 4.2.6,
Browser: Firefox 88.0.1.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
Is this identical to 'http://github.com/downthemall/downthemall/issues/285'?

**Comment by lkintact** [comment]
@RokeJulianLockhart: I don't think so - the OP of #285 was talking about the context menu (the one that's opened on right-clicking) and I was talking about the Tools menu.
